### PR TITLE
[containerapp] az containerapp service milvus: Add support for milvus vector database service

### PR DESCRIPTION
**Pull request by bgashirabake**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
containerapp

File: src/containerapp/azext_containerapp/tests/latest/test_containerapp_commands.py [hunk 1]
```diff
@@ -745,13 +749,14 @@ def test_containerapp_dev_service_binding_e2e(self, resource_group):
             JMESPathCheck('properties.template.serviceBinds[0].name', "redis"),
         ])
 
-        self.cmd('containerapp update -g {} -n {} --bind postgres:postgres_binding kafka mariadb qdrant'.format(
+        self.cmd('containerapp update -g {} -n {} --bind postgres:postgres_binding kafka mariadb qdrant milvus'.format(
```
Comment (Greedygre) [inline src/containerapp/azext_containerapp/tests/latest/test_containerapp_commands.py:752]: Please check the provisioningState.

File: src/containerapp/azext_containerapp/tests/latest/test_containerapp_commands.py [hunk 2]
```diff
@@ -730,10 +731,13 @@ def test_containerapp_dev_service_binding_e2e(self, resource_group):
 
         self.cmd('containerapp service mariadb create -g {} -n {} --environment {}'.format(
             resource_group, mariadb_ca_name, env_name))
-        
+
         self.cmd('containerapp service qdrant create -g {} -n {} --environment {}'.format(
             resource_group, qdrant_ca_name, env_name))
 
+        self.cmd('containerapp service milvus create -g {} -n {} --environment {}'.format(
```
Comment (Greedygre) [inline src/containerapp/azext_containerapp/tests/latest/test_containerapp_commands.py:738]: please check the `ProvisioningState` for creating result.

File: src/containerapp/HISTORY.rst [hunk 3]
```diff
@@ -3,6 +3,7 @@
 Release History
 ===============
 upcoming
+* 'az containerapp service': add support for creation and deletion of milvus vector database as a dev service
```
Comment (Greedygre) [inline src/containerapp/HISTORY.rst:6]: Please resolve the conflict and move the description below the +++++

**Comment by Greedygre** [comment]
please resolve the CI, rerun tests.

**Comment by Greedygre** [comment]
please resolve the conflict and fix the CI

**Pull request closed without merging**